packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Fixed an issue where `plugins install` command could fail when installing scoped package because scoped registry was used to fetch all dependencies. [#2317](https://github.com/zowe/zowe-cli/issues/2317)
```

Comment:
Can we add articles to this?: "Fixed an issue where **the** `bbbb` command could fail when when installing **a (?)** scoped package ..."